src/core/1/guide/guides/kuzzle-depth/scalability/index.md

```diff
@@ -0,0 +1,230 @@
+---
+code: false
+type: page
+title: Scalability
+---
+
+# Scalability
+
+Kuzzle can scale horizontally, provided our [official Cluster Plugin](https://github.com/kuzzleio/kuzzle-plugin-cluster) is installed.
```

Comment:
`Kuzzle can scale horizontally, provided our official Cluster Plugin is installed` ? I didn't understand the sentence

Reply:
What didn't you understand?
Is this because of the use of [provided](https://www.dictionary.com/browse/provided)?

Reply:
Yes, I never see `provided` used this way, I will sleep less stupid I suppose :p